src/Constants.ts

```diff
@@ -346,6 +349,36 @@ export class Constants {
         iconPath: {dark: "", light: ""},
         label: "Ganache Service",
         prefix: "loc",
+        type: {
+          default: {
+            label: "Regular",
+            prefix: "dfl",
+            description: "Regular",
+            isForked: false,
+            networks: {},
+          },
+          forked: {
+            label: "Forked",
```

Comment:
Thinking it might be better to go with "Fork" vs "Forked"...thoughts?

Reply:
Ok, cool!